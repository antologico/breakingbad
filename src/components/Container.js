import React from 'react'
import { Wrapper, LanguageSelectorWrapper } from '../ui'
import LanguageSelector from './LanguageSelector'

const Container = ({ children }) => <Wrapper>
    <LanguageSelectorWrapper><LanguageSelector /></LanguageSelectorWrapper>
    { children }
</Wrapper>

export default Container
