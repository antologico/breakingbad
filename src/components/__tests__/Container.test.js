import React from 'react'
import { create } from '../../tests/utils'
import Container from '../Container'

jest.mock('../LanguageSelector', () => () => <div id='selector' />)

describe('Container component', () => {
  test('renders properly', () => {
    const component = create(
        <Container>Content</Container>
    )
    expect(component.toJSON()).toMatchSnapshot()
  })
})