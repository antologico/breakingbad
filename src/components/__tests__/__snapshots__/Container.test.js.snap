// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container component renders properly 1`] = `
<div
  className="sc-dkzDqf jQOhNQ"
>
  <div
    className="sc-gsnTZi hynQrK"
  >
    <div
      id="selector"
    />
  </div>
  Content
</div>
`;
